api.ts

```diff
@@ -283,6 +285,7 @@ export interface DynamicSyncTableDef<
   getDisplayUrl: MetadataFormula;
   /** See {@link DynamicSyncTableOptions.listDynamicUrls} */
   listDynamicUrls?: MetadataFormula;
+  autocompleteCell?: MetadataFormula;
```

Comment:
I don't think this type is accurate if it accepts your new context type and not the usual parameters of an autocomplete formula.

Reply:
Yeah, I still need to fix this. I'll make a `MetadataFormulaWithoutParams` in case some other formula wants to use ExecutionContext instead of parameters similarly in the future